biogtr/inference/configs/base.yaml

```diff
@@ -0,0 +1,20 @@
+ckpt_path: "../training/models/example/example_train/epoch=0-best-val_sw_cnt=31.06133270263672.ckpt"
+tracker:
+  overlap_thresh: 0.01
+  decay_time: 0.9
+  iou: "mult"
+  max_center_dist: 1.0
+
+dataset:
+  test_dataset:
+    slp_files: ["../training/190612_110405_wt_18159111_rig2.2@11730.slp", "../training/190612_110405_wt_18159111_rig2.2@11730.slp"]
```

Comment:
Good to keep example configs, but generalize datasets + checkpoints paths.

Reply:
will prob just change this before we release it for use